app/models/primary_or_secondary.rb

```diff
@@ -1,7 +1,7 @@
 class PrimaryOrSecondary < Base
   attribute :primary_or_secondary, :string
 
-  validates :primary_or_secondary, inclusion: { in: %w(primary secondary), message: "Please answer Primary or Secondary." }
+  validates :primary_or_secondary, inclusion: { in: %w(primary secondary), message: "You must select an option" }
```

Comment:
Might be worth making your own inclusion/presence validators so you don't have to repeat the same message everywhere

Reply:
Good idea